index.js

```diff
@@ -0,0 +1,153 @@
+// import modules from other sources
+import { DateTime as localDate } from './modules/luxon.js';
+import Book from './modules/Book.js';
+import UI from './modules/ui.js';
+
+const dateTime = document.querySelector('.date-time');
+// dateTime.innerHTML = new Date().toLocaleString();
+const dt = localDate.now();
+dateTime.innerHTML = dt.toLocaleString(localDate.DATETIME_MED);
+
+const list = document.getElementById('list');
+const addList = document.getElementById('add');
+const contact = document.getElementById('contact');
+const displaySection = document.getElementById('book-list');
+const inputSection = document.querySelector('.book-input');
+const contactSection = document.querySelector('.contact-section');
+
+// eslint-disable-next-line max-classes-per-file
+
+class Collection {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Please do not disable ESLint rules unless they are in the Luxon file or are related to the number of classes per file.